backend/services/decorators.py

```diff
@@ -2,17 +2,17 @@
 from services.utils import redis
 from functools import wraps
 
+
 def validate(func):
-        @wraps(func)
-        def decorated(*args, **kwargs):
-            user_id = request.cookies.get('user_id')
-            app.logger.info(f"redis: {redis}")
-            if not user_id:
-                return {"message": "Unauthorized"}, 401
-            if not redis.exists(f"refresh_token:{user_id}"):
-                return {"message": "Unauthorized"}, 401
-            
-            return func(*args, **kwargs)
+    @wraps(func)
+    def decorated(*args, **kwargs):
+        access_token = request.cookies.get('access_token')
+        app.logger.info(f"redis: {redis}")
+        if not access_token:
+            return {"message": "Unauthorized!", "status": False}, 401
+        # if not redis.exists(f"refresh_token:{access_token}"):
+        #     return {"message": "Unauthorized"}, 401
```

Comment:
Because, we are using cookies to store the session/access token, we don't need redis as this was for the previous implementation. Github's access token doesn't expire either.